### PR TITLE
docs: review README

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Make `README` less boring.

### CODE REVIEW

**Review by mstrasinskis — APPROVED** [review]
Thank you, a great work! I added a couple of thoughts, but feel free to ignore them if you think they are not suitable.

File: BUILD.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+- [Docker](https://www.docker.com/) is a container environment. It lets you build the nns-dapp code in a sandbox, fairly reliably, and without you having to install a lot of custom tools that you may not trust. Please use [one of the official installers](https://docs.docker.com/get-docker/).
+- [Docker buildx](https://github.com/docker/buildx) is an extension that makes it easier to compile under docker. `buildx` is included in the standard docker installer for Mac desktops. If you have installed docker for an os-x server, please follow [the official guide](https://docs.docker.com/build/install-buildx/).
+- [Rosetta]() allows Mac M1 and M2 processors to run programs that use the AMD64 instruction set. If you have an M1 or M2 CPU, please:
+    - Install rosetta:
+      ```sh
+      softwareupdate --install-rosetta
+      ```
+    - Restart your laptop.
+
+#### Ubuntu Linux
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply BUILD.md:36]: What's Windows? 😉
No idea, I just refactored the content to BUILD.md

File: README.md [hunk 2]
```diff
@@ -1,155 +1,41 @@
...
-```
-
-To work on the UI locally, either use your IDE, or run the following:
-
-```sh
-cd frontend
-npm ci
-npm run dev
-```
+- Send and receive ICP
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply README.md]: Definitely

File: README.md [hunk 3]
```diff
@@ -1,155 +1,41 @@
...
+- Stake neurons
+- Vote on the governance of the Internet Computer
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply README.md]: Absolutely

**Pull request merged**